site_scons/prereq_tools/base.py

```diff
@@ -1001,6 +1019,42 @@ def load_definitions(self, **kw):
             env = self.__env.Clone()
             self.require(env, comp)
 
+    def load_defaults(self, is_arm):
+        """Setup default build parameters"""
+        #argobots is not really needed by server but it's difficult to separate
```

Comment:
argobots is needed by server, not client surely?

Reply:
Yeah, comment is wrong.  I'll fix it.